keps/sig-apps/3017-pod-healthy-policy-for-pdb/README.md

```diff
@@ -141,49 +154,51 @@ will be the current behavior.
 type PodDisruptionBudgetSpec struct {
 	
 	...
-	
-	// PodHealthyPolicy defines the criteria for when the disruption controller
-	// should consider a pod to be healthy.
-	// If no policy is specified, the legacy behavior will be used. It means
-	// only pods that are Running and Ready will be considered when the disruption
-	// controller computes "disruptionsAllowed", but all pods in the Running phase
-	// will be subject to the PDB on eviction.
-	// +optional
-	PodHealthyPolicy PodHealthyPolicy `json:"podHealthyPolicy,omitempty" protobuf:"bytes,4,opt,name=podHealthyPolicy"`
+    // UnhealthyPodEvictionPolicy defines the criteria for when unhealthy pods
+    // should be considered for eviction. Current implementation considers healthy pods,
+    // as pods that have status.conditions item with type="Ready",status="True".
+    //
+    // Valid policies are IfHealthyBudget and AlwaysAllow.
+    // If no policy is specified, the default behavior will be used,
+    // which corresponds to the IfHealthyBudget policy.
+    //
+    // Additional policies may be added in the future.
+    // Clients making eviction decisions should disallow eviction of unhealthy pods
+    // if they encounter an unrecognized policy in this field.
+    //
+    // This field is alpha-level. The eviction API uses this field when
```

Comment:
Nit: when updating this for beta, I'd just drop this information, it's important to be in the API, but not so in enhancement, and you'll have to remember to update it every time :wink: